src/components/CharactersPage/CharacterForm/CharacterForm.js

```diff
@@ -50,9 +50,17 @@ const useStyles = makeStyles((theme) => ({
   },
 }))
 
+const handlePrepareOptionsArray = (optionsArray = []) =>
+  optionsArray.map((option) => ({
+    id: option.id,
+    value: option.id,
+    label: option.name,
+  }))
```

Comment:
The only properties you need for the select object is `value` and `label`. The object id is the value